### DISCUSSION TITLE
AdoraWash Energy Total randomly set to 0

### DISCUSSION THREAD

**Discussion by danistst** [siku2/hass-vzug#41] [Device Support]
The "Energy Total" sensor of my Adora Wash V4000 is randomly set to 0. Later on, it picks up where it had been before. This messes up my Energy Dashboard.
I can also see in the logbook that "Waschmaschine Gesamtverbrauch" becomes unavailable sometimes. May this is related.

![image](https://github.com/siku2/hass-vzug/assets/67152910/c854f925-de99-4da9-bd41-6bcb07d989d3)
![image](https://github.com/siku2/hass-vzug/assets/67152910/c48e5d72-493f-4990-be64-1f5902101874)

Does anybody have a similar behaviour?

**Comment by cykohl** [comment]
I have the same issue. Could you solve the issue?

**Reply by siku2** [reply]
The issue is unfortunately on the V-Zug firmware side. It would be nice to add a work-around to the integration for this issue though.
